.bmad/bmm/docs/agents-guide.md

```
@@ -0,0 +1,1056 @@
+# BMad Method Agents Guide
+
+**Complete reference for all BMM agents, their roles, workflows, and collaboration**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)